src/common/lru.c

```diff
@@ -1,31 +1,42 @@
-/**
- * (C) Copyright 2016-2019 Intel Corporation.
+/* Copyright (C) 2016-2020 Intel Corporation
```

Comment:
why the license change?

Reply:
I just put a standart header. Should we have a different for different sources?